dbft.go

```diff
@@ -205,13 +218,27 @@ func (d *DBFT[H]) OnTimeout(height uint32, view byte) {
 		zap.Uint("view", uint(view)))
 
 	if d.IsPrimary() && !d.RequestSentOrReceived() {
-		d.sendPrepareRequest()
+		d.sendPrepareRequest(d.txSubscriptionOn || force)
 	} else if (d.IsPrimary() && d.RequestSentOrReceived()) || d.IsBackup() {
 		if d.CommitSent() || d.PreCommitSent() {
 			d.Logger.Debug("send recovery to resend commit")
 			d.sendRecoveryMessage()
 			d.changeTimer(d.timePerBlock << 1)
 		} else {
+			if d.MaxTimePerBlock != nil && d.IsBackup() {
+				if force {
+					delay := d.timePerBlock << 1
+					d.changeTimer(delay)
+					d.unsubscribeFromTransactions()
+					return
+				}
+				if !d.txSubscriptionOn && len(d.Config.GetVerified()) == 0 {
+					d.subscribeForTransactions()
+					delay := d.maxTimePerBlock<<1 - d.timePerBlock<<1
```

Comment:
@roman-khimov are you sure that it should be `2*MaxTime - 2*MinTime`? It's correct for view 1, but if we're moving to further views, then I'd say that it's more natural to take into account view number like it's done for PrepareResponse awaiting:
https://github.com/nspcc-dev/dbft/blob/7462b47e4d2d26be510f6dafdaa4832178234495/send.go#L34-L41